pylama: Import statements are in the wrong order

Symptoms:
```
# ran in venv
-> % pylama nornir
nornir/__init__.py:3:0: Import statements are in the wrong order. 'from nornir.init_nornir import InitNornir' should be before 'import pkg_resources' and in a different group. [import_order]
nornir/init_nornir.py:3:0: Import statements are in the wrong order. 'from nornir.init_nornir import InitNornir' should be before 'import pkg_resources' and in a different group. [import_order]
nornir/core/configuration.py:3:0: Import statements are in the wrong order. 'from nornir.init_nornir import InitNornir' should be before 'import pkg_resources' and in a different group. [import_order]
nornir/core/task.py:3:0: Import statements are in the wrong order. 'from nornir.init_nornir import InitNornir' should be before 'import pkg_resources' and in a different group. [import_order]
nornir/core/connections.py:3:0: Import statements are in the wrong order. 'from nornir.init_nornir import InitNornir' should be before 'import pkg_resources' and in a different group. [import_order]
nornir/core/__init__.py:3:0: Import statements are in the wrong order. 'from nornir.init_nornir import InitNornir' should be before 'import pkg_resources' and in a different group. [import_order]
```
the same happens if you do `poetry run pylama`, `make tests` or `make pylama`.
According to @carlmontanari, for some reason it does not happen in CI.
I tried different combinations of `flake8-import-order` config, including:
```
[pylama]
application-import-names = nornir
```
```
[pylama:import_order]
application-import-names = nornir
```
and some others with no success.
This seems to be working fine in `flake8`:
```
# E501 is line length 79
-> % flake8 nornir --ignore=E501
nornir/core/__init__.py:14:5: F401 'nornir.core.inventory.Host' imported but unused
```
I am giving up trying to solve this and will submit PR which disables the linter with `# noqa` on the offending line.

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -45,7 +45,6 @@ nbval = "*"
 pytest = "*"
 pytest-cov = "*"
 pylama = "*"
-flake8-import-order = "*"
 requests-mock = "*"
 black = { version = "19.10b0", allow-prereleases = true }
 mypy = "*"

--- a/setup.cfg
+++ b/setup.cfg
@@ -10,7 +10,7 @@ universal=1
 # E731: do not assign a lambda expression, use a def
 # D10?: missing docstrings
 [pylama]
-linters = mccabe,pep8,pyflakes,import_order
+linters = mccabe,pep8,pyflakes
 ignore = D203,C901
 skip = .tox/*,.venv/*
 

